tests/fixtures.ts

```diff
@@ -617,7 +655,12 @@ export class UpdateFixture extends Fixture {
                 new helios.Value(BigInt(lovelace),
                     new helios.Assets([[POLICY_ID, [[`${AssetNameLabel.LBL_000}${Buffer.from(this.handleName).toString('hex')}`, 1]]]])),
                 helios.Datum.inline(helios.UplcData.fromCbor(this.newCip68DatumCbor))
-            )
+            ),
+            // new helios.TxOutput( // pay to main address
+            //     helios.Address.fromHash(helios.PubKeyHash.fromHex(this.adminSettings[5].slice(4))),
+            //     new helios.Value(BigInt(5000000)),
+            //     helios.Datum.inline(helios.UplcData.fromCbor(this.handleCbor))
+            // ),
```

Comment:
I think this will be a required output for some of the tests. Is it added at the test level?

Reply:
I was trying to add that output on some tests.
But didn't get it done yet.
I'll push once, I make it working.